tools/cryptogenotyper/cryptogenotyper.xml

```diff
@@ -0,0 +1,135 @@
+<tool id="CryptoGenotyper" name = "CryptoGenotyper" version = "1.0">
+  <description> CryptoGenotyper is a standalone tool to analyze <i>Cryptosporidium</i> Sanger sequencing data and classify the species and subtype based on SSU rRNA and <i>gp60</i> gene markers, respectively. </description>
+  <requirements>
+    <requirement type="package" version ="1.0">cryptogenotyper</requirement>
+  </requirements>
+  <command detect_errors="exit_code">
+     <![CDATA[
+
+     #set $ref_file=''
+     #set $ref_name=''
```

Comment:
Is `$ref_name` used in the rest of the code at all?

Reply:
I removed it. I am not sure why it was added in the first place by Christine. Looks like an old artifact